src/Surfnet/StepupSelfService/SelfServiceBundle/Tests/Service/RemoteVetting/Encryption/IdentityEncrypterIntegrationTest.php

```diff
@@ -167,4 +166,17 @@ public function test_valid_private_key_required_to_decrypt()
 
         $this->assertFalse(openssl_private_decrypt($writtenData, $result, $invalidPrivateKey));
```

Comment:
I did this only to fix tests quick and dirty. Because test were failing when messages larger then a couple of hundred bytes were encrypted and I did not have a final solution at that moment and I only wanted the tests to pass.